nikola/nikola.py

```diff
@@ -564,7 +566,9 @@ def __init__(self, **config):
                                              'social_buttons_code',
                                              'search_form',
                                              'body_end',
-                                             'extra_head_data',)
+                                             'extra_head_data',
+                                             'date_format',
+                                             'js_date_format',)
```

Comment:
I’m afraid the tests fail because JS_DATE_FORMAT is not that easy, you would need special casing to do json.dumps() when the translatable context is evaluated and drop the `_GLOBAL_CONTEXT` setting.